Events/MessageEvent.cs

```diff
@@ -683,7 +685,9 @@ public static async Task MessageHandlerAsync(DiscordClient client, MockDiscordMe
                             string output = $"{Program.cfgjson.Emoji.Denied} {message.Author.Mention} was automatically warned: **{reason.Replace("`", "\\`").Replace("*", "\\*")}**\n" +
                                 $"Please consider using a Pastebin-style website or <#{Program.cfgjson.UnrestrictedEmojiChannels[0]}> to avoid punishment.";
                             DiscordMessageBuilder messageBuilder = new();
-                            messageBuilder.WithContent(output).AddComponents(button);
+                            messageBuilder.WithContent(output);
+                            if (!wasAutoModBlock)
+                                messageBuilder.AddComponents(button);
```

Comment:
Lets avoid wasteful assignments and dead data by skipping the `var button = new DiscordButtonComponent` and `Program.db` interactions as well